modules/core/test/v2/unit/trading/tradingPartner.ts

```diff
@@ -3,7 +3,8 @@ import * as nock from 'nock';
 import * as should from 'should';
 
 import fixtures from '../../fixtures/trading/tradingPartner';
-import { TradingPartnerStatus } from '../../../../src/v2/trading/tradingPartner';
+import { TradingPartnerStatus, TradingPartnerType } from '../../../../src/v2/trading/tradingPartner';
+import { TradingReferralRequesterSide } from "../../../../src/v2/trading/tradingPartners";
```

Comment:
Interesting there isn't a lint rule for single/double quotes
```suggestion
import { TradingReferralRequesterSide } from '../../../../src/v2/trading/tradingPartners';
```